asciidoctorj-api/src/main/java/org/asciidoctor/ast/Footnote.java

```diff
@@ -0,0 +1,10 @@
+package org.asciidoctor.ast;
+
+public interface Footnote {
+
+   Long getIndex();
```

Comment:
We might want to mention here that the index starts at 1 as it might be surprising.

Reply:
Will do, tx!